virttest/qemu_vm.py

```diff
@@ -2043,8 +2048,8 @@ def sort_key(dev):
                     "spice_x509_key_file", "spice_x509_cacert_file",
                     "spice_x509_key_password", "spice_x509_secure",
                     "spice_x509_cacert_subj", "spice_x509_server_subj",
-                    "spice_secure_channels", "spice_image_compression",
-                    "spice_jpeg_wan_compression",
+                    "spice_secure_channels", "spice_plaintext_channels",
```

Comment:
from man page of `qemu-kvm`:

> 
           tls-channel=[main|display|cursor|inputs|record|playback]
           plaintext-channel=[main|display|cursor|inputs|record|playback]
               Force specific channel to be used with or without TLS
               encryption.  The options can be specified multiple times to
               configure multiple channels.  The special name "default" can be
               used to set the default mode.  For channels which are not
               explicitly forced into one mode the spice client is allowed to
               pick tls/plaintext as he pleases.